hangman/interface.py

```diff
@@ -43,32 +49,127 @@ def write_instructions(self):
             "3. If the letter you guessed doesn't belong to the word, an appendage to the stickman hangman will be drawn and you will lose one unit of life."
         )
         print(
-            "4. You have 9 tries (lives) to guess the word. You lose the game when you run out of lives and the character is hung.\n\n"
+            "4. You have 9 tries (lives) to guess the word. You lose the game when you run out of lives and the character is hung."
+        )
+        self.clear()
+        print(
+            "You can quit the game at any time by pressing the key combination Ctrl + C."
         )
+        self.clear()
 
         input_flag = 0
 
         while input_flag == 0:
-            choice = input("  Enter X to return to main menu: ").upper()
+            choice = input("Enter X to return to main menu: ").upper()
 
             if choice == "X":
                 input_flag = 1
 
             else:
-                print("  Invalid input. Please try again.\n")
+                self.clear()
+                print("Invalid input. Please try again.")
+
+    def draw_hangman(self, health):
```

Comment:
this can be a staticmethod since it does not need `self`

Reply:
come to think of it, all methods in Interface should/can be static 😅, perhaps we hold off on this change and refactor the code in a new PR (low priority).